### DISCUSSION TITLE
The first number in an app-version cannot be zero or negative

### DISCUSSION THREAD

**Discussion by GrolarDan** [jreleaser/jreleaser#987] [Ideas]
Hi, 

I'm using SEMVER version pattern. In 1.1.0 there was possible to start version with zero like 0.1.3 during assemble maven goal.
But in 1.2.0 it is not possible any more. I check changes and found the change made in [fb2d119](https://github.com/jreleaser/jreleaser/commit/fb2d119fa146121b15125d43d83203f34b828aba) commit.

Based on the [SEMVER](https://semver.org) specification there are two bullets:
* A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes.
* Major version zero (0.y.z) is for initial development (*this is what I want to achieve*)

Do you think, it is possible to make it less restrictive and validate only negative number of the major part?

**Comment by aalmiray** [comment]
Oh, this os a regression and must be fixed. Sorry about that. Could you file a ticket for it? Thank you 😓

**Reply by GrolarDan** [reply]
Created issue #988
